debian/control

```diff
@@ -0,0 +1,22 @@
+Source: postgresql-ldap-sync
+Maintainer: Canonical Data Platform <data-platform@lists.launchpad.net>
+XSBC-Original-Maintainer: Debian Go Packaging Team <team+pkg-go@tracker.debian.org>
```

Comment:
You can remove the XSBC-Original-Maintainer line since this isn't going to Debian